app/templates/docs/enterprise.html

```diff
@@ -0,0 +1,625 @@
+                <li><code class="docutils literal"><span class="pre">ssoActivationFailed></code></li>
+                <li><code class="docutils literal"><span class="pre">ssoDateDelayed</span></code></li>
+                <li><code class="docutils literal"><span class="pre">idAdmins</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
+<div class="section" id="get-1-enterprises-identerprise-signupUrl">
+  <h2>GET <span class="target" id="enterprises-identerprise-signupUrl">/1/enterprises/[identerprise]/signupUrl</span><a class="headerlink" href="#get-1-enterprises-identerprise-signupUrl" title="Permalink to this headline"></a></h2>
+  <ul class="simple">
+    <li><strong>Required permissions:</strong> public</li>
+    <li><strong>Arguments</strong>
+      <ul>
+        <li><code class="docutils literal"><span class="pre">authenticate</span></code> (optional)
+          <ul>
+           <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">false</span></code></li>
+            <li><strong>Valid Values:</strong>
+              <ul>
+                <li><code class="docutils literal"><span class="pre">true</span></code></li>
+                <li><code class="docutils literal"><span class="pre">false</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li><code class="docutils literal"><span class="pre">confirmationAccepted</span></code> (optional)
+          <ul>
+           <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">false</span></code></li>
+            <li><strong>Valid Values:</strong>
+              <ul>
+                <li><code class="docutils literal"><span class="pre">true</span></code></li>
+                <li><code class="docutils literal"><span class="pre">false</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li><code class="docutils literal"><span class="pre">returnUrl</span></code> (optional)
+          <ul>
+           <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">none</span></code></li>
+            <li><strong>Valid Values:</strong>
+              <ul>
+                <li><code class="docutils literal"><span class="pre">Any valid URL</span></code></li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+      </ul>
+    </li>
+  </ul>
+</div>
+
+<div class="section" id="get-1-enterprises-identerprise-prefs">
+  <h2>GET <span class="target" id="enterprises-identerprise-prefs">/1/enterprises/[identerprise]/prefs</span><a class="headerlink" href="#get-1-enterprises-identerprise-prefs" title="Permalink to this headline"></a></h2>
+  <ul class="simple">
+    <li><strong>Required permissions:</strong> public</li>
+    <li><strong>Probably Want Examples</strong></li>
```

Comment:
Do we want this documented? It's kinda just for internal use.

It currently includes 
 - `ssoOnly` — boolean flag denoting whether the enterprise only allows members to access via SSO (i.e. not password, etc)
 - `signup` — object including `banner`, `confirmation`, `message` (markdown) and also HTML renders of the same.
 - `mandatoryTransferDate` - a date set when an enterprise is moving from not SSO-only to SSO only by which all user must “transfer their account” to the enterprise.

Especially given that we're hoping to [drop the concept of enterprise account ownership](https://trello.com/c/dotYv8J7/55-removes-the-need-for-account-ownership) I think it might be best to avoid documenting this.